samza-core/src/main/scala/org/apache/samza/system/chooser/BootstrappingChooser.scala

```diff
@@ -256,8 +256,8 @@ class BootstrappingChooser(
       null
     }
     val offsetToCheck = if (systemStreamPartitionMetadata == null) {
-      // Use null for offsetToCheck in cases where the partition metadata was 
-      // null. A null partition metadata implies that the stream is not a 
+      // Use null for offsetToCheck in cases where the partition metadata was
```

Comment:
Minor: Prefer reverting formatting changes to maintain comment history.

Reply:
It looks like IntellIj made these changes automatically. However, if I open `Annotate` in IntellIj to check the git history, the owner of this comment is not override to me even with this formatting changes. So it may be OK?

If it is not OK, I will edit the scala file without using IntellIj before updating the patch.

Reply:
If annotate works then it's fine, thanks for checking.